python/test/test_dataset.py

```diff
@@ -482,6 +492,38 @@ def test_Workspace2D_example(self):
                 spec[Data.Value, "sample1"] = np.zeros(1000)
                 spec[Data.Variance, "sample1"] = np.zeros(1000)
 
+    def test_monitors_example(self):
+        d = Dataset()
+
+        d[Coord.SpectrumNumber] = ([Dim.Spectrum], np.arange(1, 101))
+
+        # Add a (common) time-of-flight axis
+        d[Coord.Tof] = ([Dim.Tof], np.arange(9))
+
+        # Add data with uncertainties
+        d[Data.Value, "sample1"] = ([Dim.Spectrum, Dim.Tof], np.random.exponential(size=100*8).reshape([100, 8]))
+        d[Data.Variance, "sample1"] = d[Data.Value, "sample1"]
+
+        # Add event-mode beam-status monitor
+        status = Dataset()
+        status[Data.Tof] = ([Dim.Event], np.random.exponential(size=1000))
+        status[Data.PulseTime] = ([Dim.Event], np.random.exponential(size=1000))
+        d[Coord.Monitor, "beam-status"] = ([], status)
```

Comment:
What about a histogram monitor. Still treated as a nested dataset, i.e. make use of different binning?

Reply:
Yes. It can be different if required.